pkgs/tools/security/spectre-meltdown-checker/default.nix

```diff
@@ -1,38 +1,44 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper, coreutils, binutils-unwrapped }:
-
-stdenv.mkDerivation rec {
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeBinaryWrapper
+, coreutils
+, binutils-unwrapped
+}:
+
+stdenv.mkDerivation (finalAttrs: {
   pname = "spectre-meltdown-checker";
   version = "0.46";
 
   src = fetchFromGitHub {
     owner = "speed47";
     repo = "spectre-meltdown-checker";
-    rev = "v${version}";
-    sha256 = "sha256-M4ngdtp2esZ+CSqZAiAeOnKtaK8Ra+TmQfMsr5q5gkg=";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-M4ngdtp2esZ+CSqZAiAeOnKtaK8Ra+TmQfMsr5q5gkg=";
   };
 
   prePatch = ''
     substituteInPlace spectre-meltdown-checker.sh \
       --replace /bin/echo ${coreutils}/bin/echo
   '';
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeBinaryWrapper ];
 
-  installPhase = with lib; ''
+  installPhase = ''
     runHook preInstall
 
     install -Dm755 spectre-meltdown-checker.sh $out/bin/spectre-meltdown-checker
     wrapProgram $out/bin/spectre-meltdown-checker \
-      --prefix PATH : ${makeBinPath [ binutils-unwrapped ]}
+      --prefix PATH : ${lib.makeBinPath [ binutils-unwrapped ]}
 
     runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Spectre & Meltdown vulnerability/mitigation checker for Linux";
     homepage = "https://github.com/speed47/spectre-meltdown-checker";
-    license = licenses.gpl3;
-    maintainers = with maintainers; [ dotlambda ];
-    platforms = platforms.linux;
+    license = lib.licenses.gpl3;
+    maintainers = [ lib.maintainers.dotlambda ];
```

Comment:
@drupol Why not
```suggestion
    maintainers = with lib.maintainers; [ dotlambda ];
```

Reply:
I have to admit that this change is somehow opinionated.

Since this RFC: https://github.com/r-burns/nixos-rfcs/blob/rfc-inherit-as-list/rfcs/0110-inherit-as-list.md where the use of `with` is sometimes controversial, I tend to avoid using it all the times.

Reply:
Having no nested `with`s is all it takes to make static analysis possible.

Reply:
Are you ok to leave it as it is or you want me to revert the changes?

Reply:
Ok I guess it's ok since it's merged :)